diagnostic.md

```diff
@@ -6,21 +6,22 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    The router is to set up the urls to listen to and construct the aproprite patttenr to listen to. Routes and sub routes
+    The main task is `route` is retrive data from the model and prepit to be used in the template.
     ```
 
 1.  What is the command to generate a route named `boston` nested under
     `campus`?
 
     ```md
-    <!-- your response here -->
+    `ember g route boston/campus`
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+    `{{#link-to 'campus'}}Go Back{{/link-to}}`
```

Comment:
Because we are creating a nested route for the URL `/campus/boston` we would need to write it like this:
`{{#link-to 'campus.boston'}}Boston{{/link-to}}`